taxi/forms.py

```diff
@@ -0,0 +1,51 @@
+from django import forms
+from django.contrib.admin.widgets import FilteredSelectMultiple
+from django.core.validators import RegexValidator
+from django.contrib.auth.forms import UserCreationForm
+
+from taxi.models import Driver, Car
+
+
+class DriverForm(UserCreationForm):
+    license_number = forms.CharField(
+        max_length=8,
```

Comment:
why not add max length validator here and show beautiful message if validation will not go through?

Reply:
It's always better to prevent user from making mistakes, then to give him messages when mistake is already done.
I have added information message about license number parameters "
![image](https://github.com/user-attachments/assets/f6f9b956-a85e-4efc-981a-b7b8ef8e6df5)

Reply:
actually max_length, should trigger MaxLengthValidator under the hood anyway so yeah it's not really critical here